fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -507,10 +505,12 @@ public void execute(TUniqueId queryId) throws Exception {
                         LOG.warn("Analyze failed. {}", context.getQueryIdentifier(), e);
                         throw ((NereidsException) e).getException();
                     }
-                    boolean isInsertIntoCommand = parsedStmt != null && parsedStmt instanceof LogicalPlanAdapter
-                            && ((LogicalPlanAdapter) parsedStmt).getLogicalPlan() instanceof InsertIntoTableCommand;
-                    if (e instanceof NereidsException
-                            && !context.getSessionVariable().enableFallbackToOriginalPlanner && !isInsertIntoCommand) {
+                    boolean isGroupCommit = (parsedStmt != null
+                            && parsedStmt instanceof LogicalPlanAdapter
+                            && ((LogicalPlanAdapter) parsedStmt).getLogicalPlan() instanceof InsertIntoTableCommand)
+                            && !context.isTxnModel();
```

Comment:
so all insert into without txn model is group commit?

Reply:
The logic should be
```
boolean isInsertIntoCommand = (Config.wait_internal_group_commit_finish
                            || context.sessionVariable.isEnableInsertGroupCommit()) && parsedStmt != null
                            && parsedStmt instanceof LogicalPlanAdapter
                            && ((LogicalPlanAdapter) parsedStmt).getLogicalPlan() instanceof InsertIntoTableCommand;
```

but some nereids case can not pass.
I add `!context.isTxnModel()`  here becase the regression cases expect txn insert does not force fallkack

Reply:
so the variable name now should not be `isGroupCommit`? please use a better variable name, and add comments to explain it here